frontend/tests/game/play.test.js

```diff
@@ -12,11 +12,12 @@ test('Playing cards changes background as intended', () => {
     play.playCard(card);
     expect(card.style.background).toBe("green");
     
+    card = document.createElement("div");
     card.setAttribute("add-power", 1);
     play.playCard(card);
     expect(card.style.background).toBe("yellow");
 
-    card.setAttribute("target", 1);
+    card.setAttribute("target", "[:rows 0 :cards 0]");
```

Comment:
We cannot send Clojure code. Either split the information into `target-row` and `target-card` or use a json-encoded object

Reply:
The removal of `load-string` makes this change now incorrect, right?

Reply:
now it should pass the card-id

Reply:
This test only checks the card has a target, it doesn't checks if the target is correct.